### ISSUE TITLE
Different versions of scala-collection-compat pulled in transitively

### ISSUE THREAD

**Issue by andreak** [outr/scribe#482]
com.outr:scribe_2.12:jar:3.11.6
pulls in
org.scala-lang.modules:scala-collection-compat_2.12:jar:2.11.0

com.outr:moduload_2.12:jar:1.1.6
pulls in
org.scala-lang.modules:scala-collection-compat_2.12:jar:2.8.1

This should be fixed IMO.

**Comment by andreak** [comment]
Without explicitly declaring the version, using Maven.
```
            <dependency>
                <groupId>org.scala-lang.modules</groupId>
                <artifactId>scala-collection-compat_${version.scala}</artifactId>
                <version>2.11.0</version>
            </dependency>
```
results in this error:
```
[ERROR] Rule 1: org.apache.maven.enforcer.rules.dependency.DependencyConvergence failed with message:
[ERROR] Failed while enforcing releasability.
[ERROR] 
[ERROR] Dependency convergence error for org.scala-lang.modules:scala-collection-compat_2.12:jar:2.11.0 paths to dependency are:
[ERROR] +-com.visena.logger:visena-logger-scala:jar:1.3-SNAPSHOT
[ERROR]   +-com.outr:scribe_2.12:jar:3.11.6:compile
[ERROR]     +-org.scala-lang.modules:scala-collection-compat_2.12:jar:2.11.0:compile
[ERROR] and
[ERROR] +-com.visena.logger:visena-logger-scala:jar:1.3-SNAPSHOT
[ERROR]   +-com.outr:scribe_2.12:jar:3.11.6:compile
[ERROR]     +-com.outr:moduload_2.12:jar:1.1.6:compile
[ERROR]       +-org.scala-lang.modules:scala-collection-compat_2.12:jar:2.8.1:compile
```

**Comment by darkfrog26** [comment]
Good find. I need to do a new release of Moduload to update dependencies. That project is solid, so I almost never touch it anymore.

**Comment by darkfrog26** [comment]
Released `3.11.7` with an update for Moduload.
